t/Path.t

```diff
@@ -800,7 +800,11 @@ is(
         "If not provided with any paths, make_path() will return a count of 0 things created");
 }
 
-{
+ SKIP: {
+    my $skip_count = 5;
+    skip "getpwuid() and getgrgid() not implemented on Windows", $skip_count
+        if $^O eq 'MSWin32';
+
```

Comment:
Yes, your correct-win32-tests-20170419 works for me as well. I'll close this pull request.